### DISCUSSION TITLE
Can't pause, stop or hide animations

### DISCUSSION THREAD

**Discussion by jcarstairs-scottlogic** [community/community#54549] [Accessibility] [answered]
### Select Topic Area

Question

### Body

According to [WCAG 2.1 § 2.2.2](https://www.w3.org/TR/WCAG21/#pause-stop-hide), in order to meet level A of the specification, users should be able to pause, stop or hide animations which go on for longer than five seconds. But when it comes to animated GIFs inserted into conversations, issues, PRs, READMEs etc, there doesn't seem to be an obvious way of doing so.

[Here's a demo I made](https://github.com/jcarstairs-scottlogic/cannot-pause-animated-gif). (Note that clicking that link will take you to an infinitely looping animation of the Barbershop Illusion, which might cause some people migraines.) Let me know if you can't access it! (Though any infinitely repeating GIF on GitHub will suffice to prove the point.)

Am I just being daft and missing something? Or is this a genuine accessibility issue?

Note: the closest existing issues I could find are #17230 and #17228, which are also about §2.2.2 violations - but those are about the GitHub homepage, whereas this is about GIFs in conversations, issues, PRs, READMEs, etc.

**Comment by airtower-luna** [comment, marked as answer]
I see a control there, top right over the animated GIF, which works just fine:

![grafik](https://user-images.githubusercontent.com/5814919/236468088-2cef42c8-6eb6-4a21-afd0-379ab14470ec.png)

There's also this option in the settings, which worked for your GIF (it didn't play for me until I clicked the play button):

![grafik](https://user-images.githubusercontent.com/5814919/236468447-1b76bcc7-51af-4490-a329-1a5f4a961f63.png)

**Reply by jcarstairs-scottlogic** [reply]
Ah, I didn't know about that setting. Thank you!

**Comment by cehfisher** [comment]
Hi @jcarstairs-scottlogic - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/54549#discussioncomment-5815935). Could you please close this issue? Thank you! (cc @queenofcorgis)
